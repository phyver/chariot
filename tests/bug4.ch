(* constructors in pattern should be fully applied *)
data t where A : t->t

val test A = A
