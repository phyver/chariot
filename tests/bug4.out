>> option verbose is set to 1
>> inductive type t was succesfully defined
** error: the subterm A starts with a constructor that is not fully applied
